config/dev1/sys.config

```diff
@@ -19,7 +19,7 @@
       {db_path, "."},
       {persist, false},
       {autostart, false},
-      {aec_pow_cuckoo, {"lean16", "-t 5", 16}}
+      {aec_pow_cuckoo, {"lean28", "-t 5", 28}}
```

Comment:
Is this needed?

Reply:
This makes dev1 incompatible with dev2 and dev3 in tests.

Reply:
I have lean28 in my sys.config, yes... but no, this is not needed. Accidentally changed that back.